tools/spectrum-scale-driver-snap.sh

```diff
@@ -157,6 +157,11 @@ get_k8snodes_cmd="$cmd describe nodes"
 echo "$get_k8snodes_cmd"
 $get_k8snodes_cmd >> $get_k8snodes 2>&1 || :
 
+
+get_spectum_cmd="$cmd describe ds ibm-spectrum-scale-csi -n ibm-spectrum-scale-csi-driver"
```

Comment:
Not 100% consistently used in the file, but seems like there's a few variables that could be used in place of writing: `ibm-spectrum-scale-csi` and `ibm-spectrum-scale-csi-driver`.   Looks like `$PRODUCT_NAME` and `$ns`